content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -58,15 +55,17 @@ prisma.user.findMany({
 
 Will need to be changed to the following in Prisma 5:
 
-```js highlight=2;normal
+```js highlight=2-4;normal
 prisma.user.findMany({
-  OR: [{ email: 'foo@example.com' }],
+  OR: [
+    { email: 'foo@example.com' }
+  ],
 })
```

Comment:
This change to reformat the array could be applied to all examples we have if it indeed looks more obvious.

Reply:
done!